posthog/api/test/test_organization_feature_flag.py

```diff
@@ -44,3 +51,97 @@ def test_get_feature_flag_unauthorized(self):
         response = self.client.get(url)
 
         self.assertEqual(response.status_code, status.HTTP_401_UNAUTHORIZED)
+
+
+class TestOrganizationFeatureFlagCopy(APIBaseTest, QueryMatchingTest):
+    def setUp(self):
+        self.team_1 = self.team
+        self.team_2 = Team.objects.create(organization=self.organization)
+
+        self.feature_flag_key = "copied-flag-key"
+        self.feature_flag_to_copy = FeatureFlag.objects.create(
+            team=self.team_1,
+            created_by=self.user,
+            key=self.feature_flag_key,
+            filters={"groups": [{"rollout_percentage": 65}]},
+        )
+
+        super().setUp()
+
+    @snapshot_postgres_queries
+    def test_copy_feature_flag_success(self):
+        url = f"/api/organizations/{self.organization.id}/feature_flags/copy_flags"
+        data = {
+            "feature_flag_key": self.feature_flag_to_copy.key,
+            "from_project": self.feature_flag_to_copy.team_id,
+            "target_project_ids": [self.team_2.id],
+        }
+        response = self.client.post(url, data)
+
+        self.assertEqual(response.status_code, status.HTTP_200_OK)
+        self.assertIn("success", response.json())
+        self.assertIn("failed", response.json())
+
+        # copied flag is present in the response
+        self.assertEqual(self.feature_flag_key, response.json()["success"][0]["key"])
```

Comment:
For the bonus, I meant checking the full response here -> since the serializer serialises all regular fields, when a new field is added, the serializer would add it, so the test here will break.